src/cli.js

```diff
@@ -65,10 +73,31 @@ async function loadPlugin(name) {
   return plugins;
 }
 
+function findPkgJsonConfig() {
+  // fallback to original config name 'wsk' in case anything goes wrong
+  // for safe backwards compatibility
+  let cfg = 'wsk';
+
+  try {
+    // find package.json in parent directories (like yargs does)
+    const pkg = escalade(process.cwd(), (_, names) => (names.includes('package.json') ? 'package.json' : undefined));
```

Comment:
FYI I followed the yargs implementation: https://github.com/yargs/yargs/blob/0c95f9c79e1810cf9c8964fbf7d139009412f7e7/lib/yargs-factory.ts#L1611